shell/pages/c/_cluster/settings/links.vue

```diff
@@ -0,0 +1,203 @@
+import KeyValue from '@shell/components/form/KeyValue';
+import { allHash } from '@shell/utils/promise';
+
+const DEFAULT_CUSTOM_LINKS = [
+  {
+    order: 1,
+    key:   'Docs',
+    value: 'https://rancher.com/docs/rancher/v2.6/en'
+  },
+  {
+    order: 2,
+    key:   'Forums',
+    value: 'https://forums.rancher.com/'
+
+  },
+  {
+    order: 3,
+    key:   'Slack',
+    value: 'https://slack.rancher.io/'
+  },
+  {
+    order: 5,
+    key:   'Getting Started',
+    value: '/docs/getting-started'
+  }
+
+];
+
+const DEFAULT_SUPPORT_LINK = {
+  order: 4,
+  key:   'File an issue',
+  value: 'https://github.com/rancher/dashboard/issues/new'
+};
+
+const COMMUNITY_LINKS = [
+  {
+    order: 99,
+    key:   'Commercial Support',
+    value: '/support'
+  }
+];
+
+export default {
+  layout:     'authenticated',
+  components: {
+    KeyValue,
+    Loading,
+    AsyncButton,
+    Banner,
+  },
+  async fetch() {
+    try {
+      const { uiIssuesSetting, uiCommunitySetting } = await allHash({
+        uiIssuesSetting:    this.$store.dispatch('management/find', { type: MANAGEMENT.SETTING, id: SETTING.ISSUES }),
+        uiCommunitySetting: this.$store.dispatch('management/find', { type: MANAGEMENT.SETTING, id: SETTING.COMMUNITY_LINKS })
+      });
+
+      this.uiIssuesSetting = uiIssuesSetting;
+      this.uiCommunitySetting = uiCommunitySetting;
+    } catch {}
+
+    try {
+      const defaultIssueLink = { key: 'File an issue', value: this.uiIssuesSetting.value } || DEFAULT_SUPPORT_LINK;
+      const defaultLinks = [...DEFAULT_CUSTOM_LINKS, defaultIssueLink, ...COMMUNITY_LINKS];
+
+      this.uiCustomLinks = await fetchOrCreateSetting(this.$store, SETTING.UI_CUSTOM_LINKS, JSON.stringify(defaultLinks));
+
+      await this.deprecateIssueLinks();
+    } catch {}
+
+    const sValue = this.uiCustomLinks?.value || JSON.stringify(DEFAULT_CUSTOM_LINKS);
+
+    this.value = JSON.parse(sValue);
+  },
+  data() {
+    return {
+      uiCustomLinks: {},
+      bannerVal:     {},
+      value:         {},
+      errors:        [],
+    };
+  },
+  computed: {
+    mode() {
+      const schema = this.$store.getters[`management/schemaFor`](MANAGEMENT.SETTING);
+
+      return schema?.resourceMethods?.includes('PUT') ? _EDIT : _VIEW;
+    },
+
+    defaultLinks: {
+      get(issueLink) {
+        if ( issueLink ) {
+          return [...DEFAULT_CUSTOM_LINKS, issueLink, ...DEFAULT_CUSTOM_LINKS];
+        }
+
+        return [...DEFAULT_CUSTOM_LINKS, DEFAULT_SUPPORT_LINK, ...DEFAULT_CUSTOM_LINKS];
+      }
+    }
+
+  },
+  methods: {
+    useDefaults() {
+      const nonCommercialRancherLinks = this.isCommercial ? [] : COMMUNITY_LINKS;
+
+      this.value = [...DEFAULT_CUSTOM_LINKS, DEFAULT_SUPPORT_LINK, ...nonCommercialRancherLinks];
+    },
+
+    cancel() {
+      this.done();
+    },
+
+    deprecateIssueLinks() {
+      if (this.uiIssuesSetting.value || this.uiIssuesSetting.value) {
+        this.uiIssuesSetting.value = '';
```

Comment:
These values should be migrated to their new setting before they are cleared

Reply:
It happens here

https://github.com/rancher/dashboard/pull/6720/files#diff-e7649644ab782f40aabab49a017a95eeedaf9f9456abce5b5d2315db353f3a3cR73-R75